src/systemctl/systemctl.c

```diff
@@ -6029,7 +6029,7 @@ static int preset_all(int argc, char *argv[], void *userdata) {
 
         if (install_client_side()) {
                 r = unit_file_preset_all(arg_scope, arg_runtime, arg_root, arg_preset_mode, arg_force, &changes, &n_changes);
-                unit_file_dump_changes(r, "preset", changes, n_changes, arg_quiet);
+                unit_file_dump_changes(r, "preset all", changes, n_changes, arg_quiet);
```

Comment:
is this right? The log message might now show up as "Failed to preset all unit, file foobar.service already exists, …" That looks wrong. As the entries in the changes array are for individual unit files, I think "preset" was actually the right parameter to pass here...